cylc/flow/option_parsers.py

```diff
@@ -33,17 +33,26 @@ class CylcOptionParser(OptionParser):
 
     # Shared text for commands which can, & cannot, glob on cycle points:
     MULTI_USAGE_TEMPLATE = """
-TASK_GLOB is a pattern to match task proxies or task families,
-or groups of them{0}:{1}
+TASK_GLOB is a pattern to match tasks, or families, or groups of them, in
```

Comment:
(Sorry, here follows a long comment, but the aim is to save you some investigation & tweaking.)

I'm not sure if this sentence is needed now, as it seems to be covered by the more detailed descriptions for both the cycle-included & excluded globbing cases. Also the sentences are a bit mixed up.

E.g. now I get:

```console
$ cylc insert --help
Usage: cylc [control] insert [OPTIONS] TASk_GLOB [...]
...
See also 'cylc submit', for running tasks without the scheduler.

TASK_GLOB is a pattern to match tasks, or families, or groups of them, in
the task pool,
for a specific cycle point:
TASK_GLOB matches task or family names, to insert task instances into the pool
at a specific given cycle point. (NOTE this differs from other commands which
match name and cycle point patterns against instances already in the pool).
* CYCLE-POINT/TASK-NAME-GLOB
* CYCLE-POINT/FAMILY-NAME-GLOB
* TASK-NAME-GLOB.CYCLE-POINT
* FAMILY-NAME-GLOB.CYCLE-POINT

For example, to match, within the given cycle point (e.g. '20200202T0000Z'):
* all tasks: '20200202T0000Z/*' or '*.20200202T0000Z'
* all tasks named model_N for some character N: '20200202T0000Z/model_?' or
  'model_?.20200202T0000Z'
* all tasks in 'BAR' family: '20200202T0000Z/BAR' or 'BAR.20200202T0000Z'
* all tasks in 'BAR' or 'BAZ' families: '20200202T0000Z/BA[RZ]' or
  'BA[RZ].20200202T0000Z'

Arguments:
   REG                        Suite name
...
```

```console
$ cylc kill --help
Usage: cylc [control] kill [OPTIONS] REG [TASK_GLOB ...] 
...
 cylc kill REG - kill all active tasks in the suite

TASK_GLOB is a pattern to match tasks, or families, or groups of them, in
the task pool:
Multiple TASK_GLOBs can be given. They each match task proxy instances in the
current task pool by task or family name pattern, cycle point pattern, and task
state. They do NOT match any task at any point in the abstract suite graph; if
target task instances do not exist in the current pool you must insert them
first with the "cylc insert" command.
* [CYCLE-POINT-GLOB/]TASK-NAME-GLOB[:TASK-STATE]
* [CYCLE-POINT-GLOB/]FAMILY-NAME-GLOB[:TASK-STATE]
* TASK-NAME-GLOB[.CYCLE-POINT-GLOB][:TASK-STATE]
* FAMILY-NAME-GLOB[.CYCLE-POINT-GLOB][:TASK-STATE]


For example, to match:
* all tasks in a cycle: '20200202T0000Z/*' or '*.20200202T0000Z'
* all tasks in the submitted status: ':submitted'
* retrying 'foo*' tasks in 0000Z cycles: 'foo*.*0000Z:retrying' or
  '*0000Z/foo*:retrying'
* retrying tasks in 'BAR' family: '*/BAR:retrying' or 'BAR.*:retrying'
* retrying tasks in 'BAR' or 'BAZ' families: '*/BA[RZ]:retrying' or
  'BA[RZ].*:retrying'

Arguments:
   REG                           Suite name
...
```

With the following ``diff``, or something similar:

```diff
diff --git a/cylc/flow/option_parsers.py b/cylc/flow/option_parsers.py
index 614954f9f..b1bf0fecc 100644
--- a/cylc/flow/option_parsers.py
+++ b/cylc/flow/option_parsers.py
@@ -33,26 +33,25 @@ class CylcOptionParser(OptionParser):
 
     # Shared text for commands which can, & cannot, glob on cycle points:
     MULTI_USAGE_TEMPLATE = """
-TASK_GLOB is a pattern to match tasks, or families, or groups of them, in
-the task pool{0}:{1}
+{0}
 
-For example, to match{2}:{3}"""
+For example, to match:{1}"""
     # Help text either including or excluding globbing on cycle points:
     WITH_CYCLE_GLOBS = """
 Multiple TASK_GLOBs can be given. They each match task proxy instances in the
 current task pool by task or family name pattern, cycle point pattern, and task
 state. They do NOT match any task at any point in the abstract suite graph; if
 target task instances do not exist in the current pool you must insert them
-first with the "cylc insert" command.
+first with the "cylc insert" command. Valid formats are:
 * [CYCLE-POINT-GLOB/]TASK-NAME-GLOB[:TASK-STATE]
 * [CYCLE-POINT-GLOB/]FAMILY-NAME-GLOB[:TASK-STATE]
 * TASK-NAME-GLOB[.CYCLE-POINT-GLOB][:TASK-STATE]
-* FAMILY-NAME-GLOB[.CYCLE-POINT-GLOB][:TASK-STATE]
-"""
+* FAMILY-NAME-GLOB[.CYCLE-POINT-GLOB][:TASK-STATE]"""
     WITHOUT_CYCLE_GLOBS = """
 TASK_GLOB matches task or family names, to insert task instances into the pool
 at a specific given cycle point. (NOTE this differs from other commands which
 match name and cycle point patterns against instances already in the pool).
+Valid formats are:
 * CYCLE-POINT/TASK-NAME-GLOB
 * CYCLE-POINT/FAMILY-NAME-GLOB
 * TASK-NAME-GLOB.CYCLE-POINT
@@ -73,11 +72,9 @@ match name and cycle point patterns against instances already in the pool).
 * all tasks in 'BAR' or 'BAZ' families: '20200202T0000Z/BA[RZ]' or
   'BA[RZ].20200202T0000Z'"""
     MULTITASKCYCLE_USAGE = MULTI_USAGE_TEMPLATE.format(
-        "", WITH_CYCLE_GLOBS, "", WITH_CYCLE_EXAMPLES)
+        WITH_CYCLE_GLOBS.lstrip(), WITH_CYCLE_EXAMPLES)
     MULTITASK_USAGE = MULTI_USAGE_TEMPLATE.format(
-        ",\nfor a specific cycle point", WITHOUT_CYCLE_GLOBS,
-        ", within the given cycle point (e.g. '20200202T0000Z')",
-        WITHOUT_CYCLE_EXAMPLES)
+        WITHOUT_CYCLE_GLOBS.lstrip(), WITHOUT_CYCLE_EXAMPLES)
 
     def __init__(self, usage, argdoc=None, comms=False, noforce=False,
```
... it would be structured in a more logical way, I think:

```console
$ cylc insert --help
Usage: cylc [control] insert [OPTIONS] TASk_GLOB [...]

Insert new task proxies into the task pool of a running suite, e.g. to allow
...
...
See also 'cylc submit', for running tasks without the scheduler.

TASK_GLOB matches task or family names, to insert task instances into the pool
at a specific given cycle point. (NOTE this differs from other commands which
match name and cycle point patterns against instances already in the pool).
Valid formats are:
* CYCLE-POINT/TASK-NAME-GLOB
* CYCLE-POINT/FAMILY-NAME-GLOB
* TASK-NAME-GLOB.CYCLE-POINT
* FAMILY-NAME-GLOB.CYCLE-POINT

For example, to match:
* all tasks: '20200202T0000Z/*' or '*.20200202T0000Z'
* all tasks named model_N for some character N: '20200202T0000Z/model_?' or
  'model_?.20200202T0000Z'
* all tasks in 'BAR' family: '20200202T0000Z/BAR' or 'BAR.20200202T0000Z'
* all tasks in 'BAR' or 'BAZ' families: '20200202T0000Z/BA[RZ]' or
  'BA[RZ].20200202T0000Z'

Arguments:
   REG                        Suite name
...
```

```console
$ cylc kill --help
Usage: cylc [control] kill [OPTIONS] REG [TASK_GLOB ...] 

Kill jobs of active tasks and update their statuses accordingly.
 cylc kill REG TASK_GLOB ... - kill one or more active tasks
 cylc kill REG - kill all active tasks in the suite

Multiple TASK_GLOBs can be given. They each match task proxy instances in the
current task pool by task or family name pattern, cycle point pattern, and task
state. They do NOT match any task at any point in the abstract suite graph; if
target task instances do not exist in the current pool you must insert them
first with the "cylc insert" command. Valid formats are:
* [CYCLE-POINT-GLOB/]TASK-NAME-GLOB[:TASK-STATE]
* [CYCLE-POINT-GLOB/]FAMILY-NAME-GLOB[:TASK-STATE]
* TASK-NAME-GLOB[.CYCLE-POINT-GLOB][:TASK-STATE]
* FAMILY-NAME-GLOB[.CYCLE-POINT-GLOB][:TASK-STATE]

For example, to match:
* all tasks in a cycle: '20200202T0000Z/*' or '*.20200202T0000Z'
* all tasks in the submitted status: ':submitted'
* retrying 'foo*' tasks in 0000Z cycles: 'foo*.*0000Z:retrying' or
  '*0000Z/foo*:retrying'
* retrying tasks in 'BAR' family: '*/BAR:retrying' or 'BAR.*:retrying'
* retrying tasks in 'BAR' or 'BAZ' families: '*/BA[RZ]:retrying' or
  'BA[RZ].*:retrying'

Arguments:
   REG                           Suite name
...
```

Reply:
Sorry - was still fixing this - see below!